### PR TITLE
Update package after v0.2.0 release

### PR DESCRIPTION
**Pull request by rasolca**

### CODE REVIEW

**Review by albestro — APPROVED** [review]
LGTM

✅ SHA verified for 0.2.0

File: spack/packages/dla-future/package.py [hunk 1]
```diff
@@ -43,7 +44,8 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("umpire+rocm~shared", when="+rocm")
     depends_on("umpire@4.1.0:")
 
-    depends_on("pika@0.16:")
+    depends_on("pika@0.15.1:", when="@0.1.0")
```
Comment (msimberg) [inline spack/packages/dla-future/package.py:47]: Maybe
```suggestion
    depends_on("pika@0.15.1:", when="@0.1")
```
? Definitely not a blocker.

**Review by msimberg — APPROVED** [review]

**Comment by rasolca** [comment]
cscs-ci run

**Review by aurianer — APPROVED** [review]
Unrelated to this PR but is [this if](https://github.com/eth-cscs/DLA-Future/pull/964/files#diff-4ae831b914b16acdcddf955a71fcbf934d89e0cb929ed6acf161fde7f5ff41dbR174) needed if we use `define_from_variant`?

**Pull request merged**